cloud_attack_range.py

```diff
@@ -82,54 +60,149 @@
+    #simulate_parser = actions_parser.add_parser("simulate", help="Simulates attack techniques")
+    destroy_parser = actions_parser.add_parser("destroy", help="destroy attack range instances")
+    stop_parser = actions_parser.add_parser("stop", help="stops attack range instances")
+    resume_parser = actions_parser.add_parser("resume", help="resumes previously stopped attack range instances")
+    show_parser = actions_parser.add_parser("show", help="list machines")
+    # test_parser = actions_parser.add_parser("test")
+    # dump_parser = actions_parser.add_parser("dump", help="dump locally logs from attack range instances")
+    # replay_parser = actions_parser.add_parser("replay", help="replay dumps into the Splunk Enterprise server")
+
+    # Build arguments
+    build_parser.set_defaults(func=build)
+
+    # Destroy arguments
+    destroy_parser.set_defaults(func=destroy)
+
+    # Stop arguments
+    stop_parser.set_defaults(func=stop)
+
+    # Resume arguments
+    resume_parser.set_defaults(func=resume)
+
+    # Configure arguments
+    configure_parser.add_argument("-c", "--config", required=False, type=str, default='cloud_attack_range.conf',
+                                    help="provide path to write configuration to")
+    configure_parser.set_defaults(func=configure)
+
+    # Simulation arguments
+    # simulate_parser.add_argument("-st", "--simulation_technique", required=False, type=str, default="",
+    #                              help="Specify an single atomic for AWS "
+    #                                   "attack_range, example:  T1136.003, requires --simulation flag")
+    # simulate_parser.add_argument("-acf", "--attack_chain_file", required=False,
+    #                              help="attack chain file")
+    
+    # simulate_parser.add_argument("-cu", "--clean_up", required=False, type=str, default="",
+    #                              help="cleanup simulations")
+    # simulate_parser.set_defaults(func=simulate)
+
+    # # Dump  Arguments
+    # dump_parser.add_argument("-dn", "--dump_name", required=True,
+    #                          help="name for the dumped attack data")
+    # dump_parser.add_argument("--last-sim", required=False, action='store_true',
+    #                          help="overrides dumps.yml time and dumps from the start of previous simulation")
+    # dump_parser.set_defaults(func=dump)
+
+    # # Replay Arguments
+    # replay_parser.add_argument("-dn", "--dump_name", required=True,
+    #                            help="name for the dumped attack data")
+    # replay_parser.add_argument("--dump", required=False,
+    #                     help="name of the dump as defined in attack_data/dumps.yml")
+    # replay_parser.set_defaults(func=replay)
+
+    # # Test Arguments
+    # test_parser.add_argument("-tf", "--test_file", required=True,
+    #                          type=str, default="", help='test file for test command')
+    # test_parser.set_defaults(func=test)
```

Comment:
this entire section seems like it can be yanked out.

Reply:
will do in a new PR